### PR TITLE
Removed some more warnings

### PR DESCRIPTION
**Pull request by galexv**
(Ongoing issue #375)

### CODE REVIEW

File: gf/test/gf_new_tail_test_mpi.cpp [hunk 1]
```diff
@@ -34,7 +34,7 @@ class TailedGreensFunctionMPI : public ::testing::Test
   omega_sigma_gf gf2;
   typedef omega_sigma_gf gf_type;
 
-  TailedGreensFunctionMPI():beta(10), nfreq(10), nsites(5), nspins(2),
+  TailedGreensFunctionMPI():beta(10), nsites(5), nfreq(10), nspins(2),
```
Comment (egull) [inline gf/test/gf_new_tail_test_mpi.cpp]: what's the rationale of this change? (not that it does anything...)

Comment (galexv) [reply gf/test/gf_new_tail_test_mpi.cpp]: It eliminates warning about different order of declarations in the class and initializers in the ctor.
